pmd-java/src/test/java/net/sourceforge/pmd/typeresolution/MethodTypeResolutionTest.java

```diff
@@ -0,0 +1,24 @@
+package net.sourceforge.pmd.typeresolution;
```

Comment:
once this is merged I'll move this whole package in `src/test/java` to `net.sourceforge.pmd.lang.java.typeresolution` to match with `src/main/java`, but I didn't want to clutter this PR.